lib/components/narrative/narrative-itineraries.js

```diff
@@ -61,17 +161,33 @@ class NarrativeItineraries extends Component {
             useRealtime={useRealtime} />
         )
         }
-
-        {itineraries.map((itinerary, index) => {
-          return React.createElement(itineraryClass, {
-            itinerary,
-            index,
-            key: index,
-            active: index === activeItinerary,
-            routingType: 'ITINERARY',
-            ...this.props
-          })
-        })}
+        <div
+          className='list'
```

Comment:
If this is a list with children elements, then this should be an `<ul>`, and the children should be surrounded by `<li>` (with the proper styles).

Reply:
I'm not sure we're semantically that disciplined. But I'm open to being overruled. @evansiroky?

Reply:
Adding fixme for now.